js/scripts.js.php

```diff
@@ -135,21 +135,24 @@ function (response) {
       $('.plugin_formcreator_sort input[value=mostPopularSort]')[0].checked = true;
    }
 
-   fcInitMultiSelect();
-});
-
-function fcInitMultiSelect() {
    // === Add better multi-select on form configuration validators ===
    // initialize the pqSelect widget.
+      fcInitMultiSelect();
+
    $('#tabspanel + div.ui-tabs').on("tabsload", function( event, ui ) {
```

Comment:
moving the event handler out of fcInitMultiSelect() because the implemetation does not fully match the name of the function.